modules/server/src/network/aggregations/tests/aggregation.test.ts

```diff
@@ -1,27 +1,42 @@
-import { resolveAggregation } from '..';
-import { aggregation } from './fixture';
+import { AggregationAccumulator } from '../AggregationAccumulator';
+import { aggregation as fixture } from './fixture';
```

Comment:
We're being gentle today but there are a handful of 'as' which could be improved

Reply:
That's ok. Let's get them right. Can you please point out where we should improve them as I don't see any in this file?

Reply:
Oh - this is an import syntax - not TS assertion

Reply:
Sorry my bad, this is just aliasing